.kokoro

```diff
@@ -157,6 +157,75 @@ upload_bazel_test_artifacts() {
   find -L . -name "sponge_log.xml" -type f | copy_to_artifacts
 }
 
+run_bazel_affected() {
+  echo "Running bazel affected..."
+
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    move_derived_data_to_tmp
+  fi
+
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    bazel version
+    use_bazel.sh 0.20.0
+    bazel version
+  fi
+
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    select_xcode "$XCODE_VERSION"
+
+    # Move into our cloned repo
+    cd github/repo
+  fi
+
+  if [ -n "$VERBOSE_OUTPUT" ]; then
+    verbosity_args="-s"
+  fi
+
+  if [ -z "$COMMAND" ]; then
+    COMMAND="test"
+  fi
+  if [ -z "$TARGET" ]; then
+    TARGET="$(scripts/affected_targets test)"
```

Comment:
Any reason we can't pass the output of the script to the existing `run_bazel`?  There seems to be an enormous amount of overlap between the two functions.

Reply:
The intent is to replace the run_bazel job with this one, so I'd like to keep them separate for now.

Reply:
Updated the PR to consolidate the two methods for now.